### PR TITLE
Stun on self cauterization

### PR DESCRIPTION
**Pull request by NightmareChamillian**
[Balance][input wanted]
## About the PR 
This adds a short (5s) stun penalty to self cauterization.

## Why's this needed? 
Frankly, self cauterization is a little overpowered. Anyone who's smart is more than willing to trade 15 brute damage and holding still for a second for a get-out-of-jail free card with bleeding. Adding a stun to self cautery disincentivies people from using it in combat and makes it more realistic; literally burning your blood vessels closed isn't something you do and then continue with whatever task at hand prior.

In summary, this change makes people think a little bit harder as to whether or not they want to self-cauterize, and makes it more immersive when they do.

## Changelog

```changelog
(u)NightmareChamillian
(+)Self cauterization on the fly now causes a short (5 second) stun.
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
more descriptive and non-vague cl please

**Comment by NightmareChamillian** [comment]
> more descriptive and non-vague cl please

done!
